newrelic/api/opentelemetry.py

```diff
@@ -338,6 +415,9 @@ def start_span(
                     request_method = self.attributes.get("http.method")
                     request_path = self.attributes.get("http.route")
 
+                    update_GUID_flag = False if headers else True
```

Comment:
Do we really need this and the one above? Can't we just say if headers down below?